docs/contributor/storage/configuration.md

```diff
@@ -62,7 +69,37 @@ device-classes:
 ```
 Caveats:
 - Specifying lvcreate-options is at your own risk and only provided as an escape hatch by the CSI plugin. 
-- Setting spare-gb to anything other than 0 is not recommended because it does not behave predictably and usually results in more space being allocated that expected.
+- Setting spare-gb to anything other than 0 is not recommended because it does not behave predictably and usually results
+in more space being allocated that expected.
+
+#### LVM Thin Volumes
+
+Advanced storage features such as volume cloning and snapshotting are only supported on thin volumes, and thus require an
+LVM thin-pool on the host and the appropriate LVMS and cluster configuration.  For information on creating a thin-pool,
+see [OpenShift documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_logical_volumes/creating-and-managing-thin-provisioned-volumes_configuring-and-managing-logical-volumes).
```

Comment:
We can't really link to the docs behind the paywall here. Without copying that information into this repo, can we link to it in a more public way?

Reply:
@dhellmann , this page seems to be accessible without logging into the RH Customer Portal.

Reply:
OK, it still feels a little odd to link to product documentation from project documentation. Are there no other upstream project docs we can link to here, maybe for LVMO?

Reply:
Since we're referencing RHEL product documentation, I'm not sure if there is a better source. I checked our docs and we do have a few references to RHEL docs already. 

I think that's misleading here is the "OpenShift documentation" name - it should be "RHEL Documentation"

Reply:
I see your point @dhellmann.  There is OKD and K8s documentation that could be referenced instead and provide the same information.  Like Giga pointed out, there were already a couple links to docs.openshift.come, so I went with that.

Might be useful for the team to decide a concrete upstream docs source.

Reply:
Will update links to use OKD documentation.  These provide the same information as docs.openshift.com without being linked to a product.